.github/workflows/ci.yml

```diff
@@ -17,12 +17,34 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Cache - GHA
+        uses: actions/cache@v2.1.6
+        with:
+          path: ~/.cache/coursier/v1/https
+          key: ${{ runner.OS }}-coursier-cache
+
+      - name: Add extra resolvers
+        run: curl https://${{ secrets.ARTIFACTS_CACHE_BUCKET }}/resolvers.sbt --create-dirs -o ~/.sbt/resolvers.sbt
+
       - name: Setup
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
+          distribution: adopt
           java-version: 11
           java-package: jdk
           architecture: x64
 
       - name: Clean and test
         run: sbt clean test
+
```

Comment:
Do we need the commented code?

Reply:
@pperzyna ⬆️ ??

Reply:
Leave it for now, I will add it letter as a separate PR